### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
Added OobeEnableRtpAndSigUpdate

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3501

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

CC : @scanum

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -688,6 +689,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during OOBE (Out of Box experience).
+
+Valid values are:
+True -	If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
+False - (Default)	If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:698]:
```suggestion
This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during Out of Box experience (OOBE).

Valid values are:
True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```

File: docset/winserver2016-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -585,6 +586,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during OOBE (Out of Box experience).
+
+Valid values are:
+True -	If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
+False - (Default)	If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:595]:
```suggestion
True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
False (Default)	- If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```

File: docset/winserver2016-ps/defender/Set-MpPreference.md [hunk 3]
```diff
@@ -585,6 +586,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during OOBE (Out of Box experience).
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:591]:
```suggestion
This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during Out of Box experience (OOBE).
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 4]
```diff
@@ -1057,6 +1058,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during OOBE (Out of Box experience).
+
+Valid values are:
+True -	If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
+False - (Default)	If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:1067]:
```suggestion
This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during Out of Box experience (OOBE).

Valid values are:
True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: docset/winserver2016-ps/defender/Set-MpPreference.md [hunk 5]
```diff
@@ -585,6 +586,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during Out of Box experience (OOBE).
+
+Valid values are:
+True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
+False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:595]:
```suggestion
- True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
- False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 6]
```diff
@@ -688,6 +689,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during Out of Box experience (OOBE).
+
+Valid values are:
+True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
+False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:698]:
```suggestion
- True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
- False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 7]
```diff
@@ -1057,6 +1058,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OobeEnableRtpAndSigUpdate
+
+This setting allows you to configure whether real-time protection and Security Intelligence Updates are enabled during Out of Box experience (OOBE).
+
+Valid values are:
+True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
+False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:1067]:
```suggestion
- True - If you enable this setting, real-time protection and Security Intelligence Updates are enabled during OOBE.
- False (Default) - If you either disable or don't configure this setting, real-time protection and Security Intelligence Updates during OOBE aren't enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, some more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
Hi @tiburd hope all is well. Could you please help with this PR? Thanks!

**Pull request merged**